templates/admin.php

```diff
@@ -33,11 +33,11 @@
     <p><?php p($l->t("This setting will enable the theme by default, for any unauthenticated users and users who haven't set a preference.")); ?></p>
     <input type="checkbox" class="checkbox" id="breezedark-theme-enabled" <?php p($themeEnabled ? "checked" : ""); ?>>
     <label for="breezedark-theme-enabled"><?php p($l->t("Enable Breeze Dark theme by default")); ?></label>
-    <p><?php p($l->t("This setting will allow you to choose if the login page should be themed when the theme is enabled by default")); ?></p>
+    <p><?php p($l->t("This setting will allow you to choose if the login page should be themed when the theme is enabled by default.")); ?></p>
     <input type="checkbox" class="checkbox" id="breezedark-theme-login-page" <?php p($themeEnabled ? "" : "disabled");?> <?php p($themeLoginPage ? "checked" : "");?>>
     <label for="breezedark-theme-login-page"><?php p($l->t("Theme the login page")); ?></label>
 
     <h3><?php p($l->t("Custom Styling")); ?></h3>
-    <textarea id="breezedark-theme-custom-styling" style="width: 100%; height: 200px; font-family: monospace;" placeholder="Insert custom styling here..."><?php p($themeCustomStyling); ?></textarea>
-    <button id="breezedark-theme-custom-styling-button">Save</button>
+    <textarea id="breezedark-theme-custom-styling" style="width: 100%; height: 200px; font-family: monospace;" placeholder="<?php p($l->t("Insert custom styling here …")); ?>"><?php p($themeCustomStyling); ?></textarea>
```

Comment:
Is the space between ```here``` and ```…``` in ```"Insert custom styling here …"``` intentional? You aren't using it in the polish translation so I figure it might be a typo.

Reply:
This is intentional. It is assumed that the correct English language is separated by an ellipse with a space. However, in Polish it is without a space character.

Reply:
Okay, thank you for the clarification.